torch/_dynamo/utils.py

```diff
@@ -517,7 +517,6 @@ def is_function(value):
         value,
         (
             types.FunctionType,
-            types.MethodType,
```

Comment:
I think this is typo in my previous PR, general method (non-nn.module) should be wrapped as ```UserDefinedObjectVariable``` as the binding object matters.